packages/material-ui/src/ListItemText/ListItemText.js

```diff
@@ -5,109 +5,91 @@ import withStyles from '../styles/withStyles';
 import Typography from '../Typography';
 import ListContext from '../List/ListContext';
 
-export const styles = theme => ({
+export const styles = {
   /* Styles applied to the root element. */
   root: {
     flex: '1 1 auto',
     minWidth: 0,
-    padding: '0 16px',
```

Comment:
@oliviertassinari We are migrating v3 -> v4 and here and there paddings get removed breaking some layout 
is there any particular reason why these changes were applied?
Do we know how many components undergone such style adjustments?

Reply:
I don't remember, nor I have the time to look at it, we encourage to have visual regression tests when upgrading